python/resdata/grid/cell.py

```diff
@@ -81,7 +81,7 @@ def __eq__(self, other):
 
     def __neq__(self, other):
         if isinstance(other, Cell):
-            return not self == other
```

Comment:
I think this becomes circular as `!=` is implemented by `__neq__`. At least that is how it used to be. I think the inclusion of `__neq__` was only for python 2 and not necessary in python 3. This should probably be tested before we make the change.